1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
+
+ENV GOLANG_VERSION 1.6.1
+ENV GOLANG_DOWNLOAD_URL "https://golang.org/dl/go$GOLANG_VERSION.windows-amd64.zip"
+
+RUN powershell -Command \
+	(New-Object System.Net.WebClient).DownloadFile('%GOLANG_DOWNLOAD_URL%', 'go.zip') ; \
+	Expand-Archive go.zip -DestinationPath c:\\ ; \
+	Remove-Item go.zip -Force
+
+RUN setx /M PATH "C:\go\bin;%PATH%" && \
```

Comment:
How? It's the same number of bits over the wire (roughly), and it makes better use of caching because (for example) the `RUN setx` command (and subsequent ones) can be changed without having to re-run the the download and un-archive